package.json

```diff
@@ -66,6 +66,7 @@
     "recast": "^0.23.4",
     "ts-jest": "^29.1.1",
     "ts-node": "^10.9.1",
+    "tsc-hooks": "^1.1.2",
```

Comment:
`tsc` does not copy any metadata file that is not directly referenced as an `import`, this package helps with that.

Reply:
👏